### DISCUSSION TITLE
检测框很靠近文字，导致识别效果不好

### DISCUSSION THREAD

**Discussion by mrlihellohorld** [PaddlePaddle/PaddleOCR#12838] [Q&A]
![pdf_3](https://github.com/PaddlePaddle/PaddleOCR/assets/31719207/7cd0ba4e-b4b1-450b-b42c-2887033c1157)
我发现检测框很靠近文字边缘，导致于我用自己训练的crnn文字识别模型效果不好。
如果把检测模型替换为craft,效果好一些：
![pdf_3](https://github.com/PaddlePaddle/PaddleOCR/assets/31719207/f521dcce-8111-4418-92c2-3242c17a8bd3)

**Comment by 07571026** [comment]
如果你原本的檢測模型是使用DB的話，他有一個參數是unclip_ratio，會影響後處理繪製檢測框的大小，2.7版的預設是2.0，數值越大框就越大，反之亦然，你可以試著調整看看

如果使用command執行程式的話，它的argument是 --det_db_unclip_ratio
如果要直接在.py中修改的話，請到ppocr/postprocess/db_postprocess.py [傳送門](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/ppocr/postprocess/db_postprocess.py) 找 unclip_ratio

**Comment by MarginGitHub** [comment]
最简单的就是在四周padding空白区域
